examples/fuse_examples/multimodality/ehr_transformer/dataset.py

```diff
@@ -194,13 +195,13 @@ class OpMapToCategorical(OpBase):
       (third bin in calculated distribution)
     """
 
-    def __call__(self, sample_dict, bins: dict) -> Any:
+    def __call__(self, sample_dict: NDict, bins: dict) -> Any:
 
         # convert continuous measurements to categorical ones based
         # on defined bins mapping static clinical characteristics
         # (Age, Gender, ICU type, Height, etc)
         for k in sample_dict["StaticDetails"]:
-            sample_dict["StaticDetails"][k] = k + "_" + str(np.digitize(sample_dict["StaticDetails"][k], bins[k]))
+            sample_dict[f"StaticDetails.{k}"] = k + "_" + str(np.digitize(sample_dict["StaticDetails"][k], bins[k]))
```

Comment:
A small fix to match the new `NDict` impl.

Since we now don't return a "real" nested dict, we can't change a returned sub-dict and expect the changes to be reflected in the original dictionary. 


This kind of fix might be needed to be applied on other projects that are not covered by the CI tests.